.github/workflows/ode-toolbox-build.yml

```diff
@@ -35,14 +35,15 @@ jobs:
       - name: Install Python dependencies
         run: |
           python -m pip install --upgrade pip pytest pycodestyle codecov pytest-cov wheel
-          python -m pip install -r requirements.txt
+          python -m pip install numpy
           if [ "${{ matrix.with_gsl }}" == "1" ]; then python -m pip install pygsl ; fi
+          python -m pip install -r requirements.txt
           export PYTHON_VERSION=`python -c "import sys; print('.'.join(map(str, [sys.version_info.major, sys.version_info.minor])))"`
           echo "Python version detected:"
           echo $PYTHON_VERSION
 
           pip uninstall --yes sympy
-          pip install sympy${{ matrix.sympy_version }}
+          pip install --upgrade sympy${{ matrix.sympy_version }}
```

Comment:
Why is the `--upgrade` flag required here?

Reply:
Flag removed.